src/not-found-page/NotFoundPage.tsx

```diff
@@ -0,0 +1,18 @@
+import { Container } from '@openedx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import messages from './messages';
+
+const NotFoundPage = () => {
+  const intl = useIntl();
+
+  return (
+    <Container size="sm">
+      <p className="my-0 py-5 text-muted text-center not-found-message mx-auto">
```

Comment:
This class doesn't exist now that `NotFoundPage.scss` was removed in https://github.com/openedx/frontend-app-catalog/compare/a7c8e0155c756d33555f4655f3e24ba15ab88d64..46017bd5b0a2e8a0df64132aa38ee9d6d958cdd3

```suggestion
      <p className="my-0 py-5 text-muted text-center mx-auto">
```